documentation/specs/src/base-ledger/governance.md

```diff
@@ -73,33 +86,68 @@ The governance machinery also relies on a subkey stored under the `NAM` token ad
 This is to leverage the `NAM` VP to check that the funds were correctly locked.
 The governance subkey, `/\$GovernanceAddress/proposal/\$id/funds` will be used after the tally step to know the exact amount of tokens to refund or move to Treasury.
 
+### Supported proposal types
+
+At the moment, Namada supports 3 types of governance proposals:
+
+```rust
+pub enum ProposalType {
+  /// Carries the optional proposal code path
+  Custom(Option<String>),
+  PGFCouncil,
+  ETHBridge,
+}
+```
+
+`Custom` represents a generic proposal with the following properties:
+
+- Can carry a wasm code to be executed in case the proposal passes
+- Allows both validators and delegators to vote
+- Requires 2/3 of the total voting power to succeed
+- Doesn't expect any memo attached to the votes
+
+`PGFCouncil` is a specific proposal to elect the council for _Public Goods Funding_:
+
+- Doesn't carry any wasm code
+- Allows both validators and delegators to vote
+- Requires 1/3 of the total voting power to vote for the same council
+- Expect every vote to carry a memo in the form of a tuple `(Vec<Address>, BudgetCap)`
```

Comment:
yes but people can vote for multiple councils & budget caps (approval voting) so it should be a set of tuples, right?